symfony/routing/4.2/config/routes.yaml

```diff
@@ -0,0 +1,3 @@
+#index:
+#    path: /
+#    controller: App\Controller\DefaultController::index
```

Comment:
Shouldn't this be just one colon?

Reply:
didn't we deprecate the single colon syntax? this is the same file as for lower versions btw

Reply:
same as in the other PR: I did confuse what we deprecate 🙈